app/email.go

```diff
@@ -297,7 +297,9 @@ func (a *App) SendInviteEmails(team *model.Team, senderName string, senderUserId
 			mlog.String("retry_after", result.RetryAfter.String()),
 			mlog.Err(err))
 		return
-	} else if err != nil {
+	}
+
+	if err != nil {
```

Comment:
Idiomatically, should we check `err != nil` before looking at `rateLimited`?

Reply:
Yes, good point :)

Reply:
Fixed :)